index.html

```diff
@@ -4,10 +4,30 @@
 		<meta charset="UTF-8" />
 		<link rel="icon" type="image/svg+xml" href="/images/communityFocus.png" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta name="description" content="Create a shareable timer for you and your friends!" />
+
 		<title>Community Focus</title>
+
```

Comment:
maybe even add something like this
```suggestion
    <meta name="description" content="Create a shareable timer for you and your friends!">
    <meta name="image"
      content="https://communityfocus.app/images/communityFocus.png">
```